translation/source/site.xml

```diff
@@ -271,7 +271,6 @@
 		<item quantity="other">Give %s seconds</item>
 	</plurals>
 	<string name="thisAccountViolatedTos">This account violated the Lishogi Terms of Service</string>
-	<string name="thisPlayerArtificiallyIncreasesTheirRating">This player artificially increases/decreases their rating</string>
```

Comment:
Might want to remove this line in the `dest` folder to fix the Lint Translation errors ( https://github.com/WandererXII/lishogi/pull/417/checks?check_run_id=3523201281 ). Also I think it would be better to mention that it has been taken from a pull request sent to Lila. :)

Reply:
Ok Thanks for the review!

Reply:
> Might want to remove this line in the `dest` folder to fix the Lint Translation errors ( https://github.com/WandererXII/lishogi/pull/417/checks?check_run_id=3523201281 ). Also I think it would be better to mention that it has been taken from a pull request sent to Lila. :)

This is also the reason I don’t think https://github.com/WandererXII/lishogi/pull/416 makes sense.